### PR TITLE
merge: develop to qa

### PR DESCRIPTION
**Pull request by tipusinghaw**
- Fixed set primary did issue
- Added queue in retry bulk-issuance
- W3C support for bulk-issunace
- Restrict user to create max 10 organization

### CODE REVIEW

**Review by pranalidhanavade — APPROVED** [review]
LG

**Review by KulkarniShashank — APPROVED** [review]
LG

File: apps/issuance/src/issuance.service.ts [hunk 1]
```diff
@@ -1018,165 +1043,172 @@ async sendEmailForCredentialOffer(sendEmailCredentialOffer: SendEmailCredentialO
...
-      name?: string;
-      upload_type: string;
-      status: string;
-      orgId: string | number;
-      createDateTime: Date;
-    } = {
+    if ('' === requestId.trim()) {
+      throw new BadRequestException(ResponseMessages.issuance.error.missingRequestId);
+    }
+    // let credentialType : SchemaType;
```
Comment (GHkrishna) [inline apps/issuance/src/issuance.service.ts:1049]: In future we need to remove this commented code, if not required

File: libs/prisma-service/prisma/schema.prisma [hunk 2]
```diff
@@ -473,6 +473,7 @@ model file_upload {
   file_data           file_data[]
   organisation        organisation? @relation(fields: [orgId], references: [id])
   orgId               String?       @db.Uuid
+  credential_type     String?
```
Comment (GHkrishna) [inline libs/prisma-service/prisma/schema.prisma:476]: We can look if prisma allows enums while storing data in DB

**Review by GHkrishna — APPROVED** [review]
LGTM

**Review by bhavanakarwade — APPROVED** [review]

**Pull request merged**